package/MDAnalysis/analysis/contacts.py

```diff
@@ -438,6 +438,8 @@ def __init__(self, u, select, refgroup, method="hard_cut", radius=4.5,
                 self.r0.append(distance_array(refA.positions, refB.positions))
                 self.initial_contacts.append(contact_matrix(self.r0[-1],
                                                             radius))
+        # Number of initial contacts present
+        self.n_initial_contacts=self.initial_contacts[0].sum()
```

Comment:
As mentioned by @lilyminium in #2604 you'll want to document this in the docstring as an attribute that users should be aware of. You'll also want to document things like the `r0` attributes too.

Also please do keep to PEP8 convention, you should have spaces around the `=` here.

Reply:
Thanks a lot for reverting @IAlibay, I added the changes suggested by you. Please let me know if its okay. You're right about testing.